tests/e2e/api/faucet.test.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Aptos } from "../../../src/api/aptos";
+import { AptosConfig } from "../../../src/api/aptos_config";
+import { Account } from "../../../src/core/account";
+import { SigningScheme } from "../../../src/types";
+import { Network } from "../../../src/utils/apiEndpoints";
+
+describe("Faucet", () => {
+  test("it should fund an account", async () => {
+    const config = new AptosConfig({ network: Network.LOCAL });
+    const aptos = new Aptos(config);
+    const testAccount = Account.generate({ scheme: SigningScheme.Ed25519 });
+
+    // Fund the account
+    await aptos.fundAccount({ accountAddress: testAccount.accountAddress.toString(), amount: 10_000_000 });
```

Comment:
Yeah, this I see is why you don't want to pass the string, and instead the account address, we'll have to review this whole input strategy afterwards

Reply:
Yep. Including the args as input param on some places, we can revisit after Alpha :)